app/views/static_pages/directory.html.erb

```diff
@@ -49,7 +46,7 @@
       <div class="two">No results. Try searching for something else.</div>
      <% else %>
       <% @results.each do |person| %>
-       <p> <font size="5">
+       <p style="font-size: x-large"> 
```

Comment:
This is an improvement, but we should probably use a stylesheet for this font size, and use [rem units](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/).